### ISSUE TITLE
rhs module load error - testing on Windows

### ISSUE THREAD

**Issue by ajgpitch** [qutip/qutip#728]
Specifically this test `test_interpolate.test_interpolate_brevolve3()` followed by `test_mcsolve.test_MCCollapseTimesOperators` generates an error or the form
```
Process SpawnPoolWorker-1:
Traceback (most recent call last):
  File "C:\Anaconda2\envs\q4_2-py35\lib\multiprocessing\process.py", line 249, in _bootstrap
    self.run()
  File "C:\Anaconda2\envs\q4_2-py35\lib\multiprocessing\process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Anaconda2\envs\q4_2-py35\lib\multiprocessing\pool.py", line 108, in worker
    task = get()
  File "C:\Anaconda2\envs\q4_2-py35\lib\multiprocessing\queues.py", line 345, in get
    return ForkingPickler.loads(res)
ImportError: No module named 'rhs47600'
```
The rhs... file in generated by the brevolve3 test. The test appears to finish fine. But somehow it looks like some process is hanging around. Why this is triggered by the MCCollapseTimesOperators test is a bit of a mystery.

**Comment by ajgpitch** [comment]
The issue occurs when running tests on Windows. Both p35 and py36 behave the same. Both tests work fine when run separately

**Comment by ajgpitch** [comment]
This will repeat the issue quite quickly and always reliability
```
import traceback
import sys
import qutip.tests.test_interpolate as test_interpolate
import qutip.tests.test_mcsolve as test_mcsolve

def main():

    try:
        test_interpolate.test_interpolate_brevolve3()
        test_mcsolve.test_MCCollapseTimesOperators()

        print("Tests passed")
    except:
        print("Test failed:")
        traceback.print_exc()

if __name__ == "__main__":
    main()
```

The main function is necessary on windows for scripts that use multiprocessing somewhere

**Comment by nonhermitian** [comment]
I am guessing that there is some issue in the MC solve function. The test that you do should not be looking for a RHS file as the test is time-independent.  Why this only happens on Win is an interesting question.  I put up a fork that may work here:

https://github.com/nonhermitian/qutip/tree/config_reset_test

Let me know how it goes.  It seemed to work on my Win7 VM.  It basically removes the old td file everytime it is not needed.  I am not sure why we did not do this before.  It still needs to be checked that the rhs_reuse setting in the Options class still works.

**Comment by ajgpitch** [comment]
Fixed by #729
